web/src/hooks/UseEvents.tsx

```diff
@@ -89,7 +89,7 @@ export function useEvents() {
       type: NotificationEventType.imageExpiry,
       title: 'Image expiry trigger',
       icon: <HistoryIcon />,
-      enabled: true,
+      enabled: config?.features.IMAGE_EXPIRY_TRIGGER,
```

Comment:
you'll have to add `FEATURE_IMAGE_EXPIRY_TRIGGER` to `CLIENT_WHITELIST` in config.py

Reply:
yes, missed that.